web/packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/Databases.tsx

```diff
@@ -55,7 +56,13 @@ function DatabaseList(props: State) {
   } = props;
   const dbs = fetchAttempt.data?.agentsList.map(makeDatabase) || [];
   const disabled = fetchAttempt.status === 'processing';
-  const emptyText = getEmptyTableText(fetchAttempt.status, 'databases');
+  const loggedInUser = useWorkspaceLoggedInUser();
+  const emptyText = getEmptyTableText(
+    fetchAttempt.status,
+    'databases',
+    agentFilter.search || agentFilter.query,
+    loggedInUser?.acl?.tokens.create
+  );
```

Comment:
Arguably this could be refactored, e.g. to use an argument object instead of positional args, and perhaps even extracted somehow as to not having to be repeated three times. It also could be argued that checking for no resources should be done within some kind of a switch with cases over `attempt.status`, instead of assuming that empty text for "success" attempt === empty text for no records and using the term "empty text" to show errors and loading state.

However, since we're moving to unified resources anyway, I decided to not change it too much.